frontend/src/components/Psychologist/Patients/AddEditPatient.jsx

```diff
@@ -49,6 +49,7 @@ const AddEditPatient = () => {
   const pageTitle = patientId ? 'Modifier un patient' : 'Nouveau patient';
   const intro = patientId
     ? 'Modifiez les informations de l\'étudiant.'
+    // eslint-disable-next-line max-len
```

Comment:
Why ?

Reply:
Cause es-lint was saying that this line was too long.

Reply:
but you didn't touch it...

Reply:
Yeah, i don't know why it's displaying this. I deleted it.